Makefile.core.mk

```diff
@@ -85,6 +85,9 @@ clean:
 lint: clean_public build_nominify lint-copyright-banner lint-python lint-yaml lint-dockerfiles lint-scripts lint-sass lint-typescript lint-go
 	@scripts/lint_site.sh
 
+lint-fast:
```

Comment:
Can we change the name to `lint-no-link` or something similar?

Removing the link checker will make it faster, but it's still not real fast. And `fast` doesn't really tell me what is happening.

Reply:
Its sort of declarative vs imperative. If we make it `lint-no-link` then dev has to wonder "why do I not want link checking", we cannot add more "fast linters" in the future, its not actually accurate since it skips other things as well, etc. I think its also a fairly standard practice to have "fast" tests and "full" tests (maybe just in Google).

I am not the target audience for this so I can change it to whatever is desired, just my thought process here